main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"github.com/wmaldonadoc/academy-go-q42021/infrastructure/datastore"
+	"github.com/wmaldonadoc/academy-go-q42021/infrastructure/router"
+	"github.com/wmaldonadoc/academy-go-q42021/registry"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func bootingGlobalLogger() *zap.Logger {
+	// Replacing logger global
+	config := zap.NewDevelopmentConfig()
+	config.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
+	config.EncoderConfig.TimeKey = "timestamp"
+	config.EncoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	logger, _ := config.Build()
+	return logger
+
```

Comment:
Best practices tell us that code blocks should never end in an empty line, and returns should be in a separated block.

```suggestion

	return logger
```